script/travis-install-linux

```diff
@@ -1,5 +0,0 @@
-#!/usr/bin/env bash
-set -e
-
-# See: https://github.com/kylef/swiftenv/wiki/Travis-CI
-curl -sL https://gist.github.com/kylef/5c0475ff02b7c7671d2a/raw/621ef9b29bbb852fdfd2e10ed147b321d792c1e4/swiftenv-install.sh | bash
```

Comment:
This file is still used here: https://github.com/Quick/Quick/blob/9767dfd0016f4654af495bb938f198d14eb8fa9d/.travis.yml#L22

Reply:
@ikesyo - Oh gosh, I am so sorry! I've restored the file in my latest commit.